### PR TITLE
Update MUI Theme Palette 

### PR DESCRIPTION
**Pull request by teenoh**
## 🌟 What is the purpose of this PR?

This updates default MUI palette fields so that MUI components are closer to our theme system out of the box

### 📜 Does this require a change to the docs?

- No

## 🔗 Related links

- [Override MUI default palette fields](https://app.asana.com/0/1201864704093982/1201891760623780) (_internal_)

## 🛡 What tests cover this?

-

### CODE REVIEW

**Comment by akash-joshi** [comment]
@teenoh sorry about the broken CI. This should help you out: https://github.com/hashintel/hash/pull/378

File: packages/hash/frontend/src/theme/palette.ts [hunk 1]
```diff
@@ -149,10 +149,45 @@ export const customColors = {
...
+  },
+  secondary: {
+    dark: customColors.purple[90],
+    main: customColors.purple[70],
+    light: customColors.purple[50],
+  },
+  warning: {
+    dark: customColors.blue[80],
+    main: customColors.blue[60],
+    light: customColors.blue[40],
```
Comment (benwerner01) [inline packages/hash/frontend/src/theme/palette.ts:166]: Shouldn't warnings be orange?
```suggestion
    dark: customColors.orange[80],
    main: customColors.orange[60],
    light: customColors.orange[40],
```

**Review by benwerner01 — CHANGES_REQUESTED** [review]
Thank you Teeno, looks good 👍 

I think I spotted a typo in one of the colours, once that's addressed this is good to merge

File: packages/hash/frontend/src/theme/palette.ts (hunk 1 above)
Comment (teenoh) [reply packages/hash/frontend/src/theme/palette.ts]: Not sure how I missed this. Thanks for catching! Fixed here => 439e6aa

**Review by benwerner01 — APPROVED** [review]
Thanks Teeno 🙂

**Pull request merged**